web/components/three.js/Warehouse.tsx

```diff
@@ -0,0 +1,20 @@
+import { Canvas } from "@react-three/fiber";
+import { OrbitControls, PerspectiveCamera } from "@react-three/drei";
+import Floor from "./Floor";
+import "./ThreeStyle.css"
+
+export default function Warehouse() {
+	return (
+		<div id="canvas-container">
+			<Canvas
+				className={"styledcanvas"}
```

Comment:
Non segue la sintassi di Tailwind. 
styledcanvas ha come unico attributo height: 100vh
In tailwind si scrive className={"h-screen"}

Reply:
grazie, non avevo capito evidentemente cos'era tailwind